File: app/src/main/java/com/yunzia/hyperstar/hook/init/SystemUIHookForOS1.java
package com.yunzia.hyperstar.hook.init;

import static de.robv.android.xposed.XposedHelpers.findAndHookMethod;

import android.content.res.XModuleResources;

import com.yunzia.hyperstar.hook.app.systemui.os1.NavigationBarBackground;
import com.yunzia.hyperstar.hook.base.Init;
import com.yunzia.hyperstar.hook.base.InitHooker;

import de.robv.android.xposed.callbacks.XC_InitPackageResources;

@Init(packageName = "com.android.systemui")
public class SystemUIHookForOS1 extends InitHooker {

    private final PluginHookForOS1 pluginHookForOS1 = new PluginHookForOS1();


    @Override
    public void initResources() {
        initResource(pluginHookForOS1);
        if (!resparam.packageName.equals(mPackageName)) return;
    }

    @Override
    public void initHook() {
        initHooker(pluginHookForOS1);
        initHooker(new NavigationBarBackground());

        doTestHook();
    }


    private void doTestHook() {



    }


}

File: app/src/main/java/com/yunzia/hyperstar/hook/init/SystemUIHookForOS1.kt
package com.yunzia.hyperstar.hook.init

//import com.yunzia.annotations.Init
import com.yunzia.hyperstar.hook.app.systemui.os1.LowDeviceBackgroundColor
import com.yunzia.hyperstar.hook.app.systemui.os1.SystemBarBackground
import com.yunzia.hyperstar.hook.core.base.BaseHooks
import com.yunzia.hyperstar.hook.core.annotation.Init

@Init(packageName = "com.android.systemui", versions = [1])
object SystemUIHookForOS1 : BaseHooks() {

    override fun init() {
        initHooks(
            PluginHookForOS1,
            SystemBarBackground,
            LowDeviceBackgroundColor
        )
    }
}